ferry/database/models.py

```diff
@@ -79,6 +79,52 @@ class Season(BaseModel):
     ),
 )
 
+# Similar courses with FastText
+course_fasttext_similars = Table(
+    "fasttext_similars_staged",
+    Base.metadata,
+    Column(
+        "source",
+        ForeignKey("courses_staged.course_id"),
+        primary_key=True,
+        index=True,
+    ),
+    Column(
+        "target",
+        ForeignKey("courses_staged.course_id"),
+        primary_key=True,
```

Comment:
whoa I had no idea you could have multiple fields in your primary key

Reply:
Yep, apparently it's called a [composite key](https://en.wikipedia.org/wiki/Composite_key).